src/tel_download.js

```diff
@@ -345,38 +379,35 @@
   // For webk /k/ webapp
   setInterval(() => {
     /* Voice Message */
-    const voiceMessages = document.querySelectorAll("audio-element");
+    const pinnedAudio = document.body.querySelector(".pinned-audio");
+    let dataMid;
+    let downloadButtonPinnedAudio = document.body.querySelector("._tel_download_button_pinned_container") || document.createElement("button");
+    if (pinnedAudio) {
```

Comment:
here and below on line 406, the button will constantly updating itself for every tick even though nothing has changed on the page, and thus the button is kinda "blinking". Could you update the logic here to avoid update if the button is already there and nothing is required to update? 

It could be something like:
```suggestion
    if (link && button_is_not_present)) {
                      dataMid = pinnedAudio.getAttribute("data-mid");
      downloadButtonPinnedAudio.className = "btn-icon tgico-download _tel_download_button_pinned_container";
      downloadButtonPinnedAudio.innerHTML =
          '<span class="tgico button-icon">\uE93D</span>';
    }
```

![2023-09-26 12 21 21](https://github.com/Neet-Nestor/Telegram-Media-Downloader/assets/23090573/e0237be0-2fb6-4ed1-acfa-45b6452b0f51)